ui/apps/platform/src/Containers/NetworkGraph/namespace/NamespaceDeployments.tsx

```diff
@@ -23,8 +24,18 @@ const deployments = [
 ];
 
 function NamespaceDeployments() {
+    const [value, setValue] = React.useState('');
     const { page, perPage, onSetPage, onPerPageSelect } = usePagination();
 
+    const onChange = (newValue: string) => {
+        setValue(newValue);
+    };
+
+    // @TODO: This will be replaced by filtering on the backend through the API
```

Comment:
Could you explain that a bit more? I was confused about what you meant by deferring the data from the URL state. I was thinking that this component would likely have a prop like `namespace` or `namespaceName` and we could use that to query for a list of deployments for that namespace. Either that or we can have another prop for `deployments` and maybe just reuse the deployments we see in the network graph, since all that info is already there.

Reply:
I think I misunderstood the TODO. I thought you meant to re-call the API when the user filtered the table in the sidebar, and I thought we said we would not do that, at least initially.